pkg/storaged/jobs.js

```diff
@@ -184,7 +187,13 @@
                 var j = job(path);
 
                 var age_ms = server_now - j.StartTime/1000;
-                return age_ms >= 2000;
+                if (age_ms >= 2000)
+                    return true;
+
+                if (j.ExpectedEndTime > 0 && (j.ExpectedEndTime/1000 - server_now) >= 2000)
```

Comment:
Isn't checking `j.ExpectedEndTime > 0` redundant here? If that were `<= 0`, the second expression would always evaluate to false.